dotnet/Razorvine.Pyrolite/Benchmarks/Benchmarks.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <LangVersion>7</LangVersion>
```

Comment:
Isn't 7 the default? I think you can remove this line.

Reply:
6 is set for other projects in this repo, but BDN does not support it (the auto-generated code uses C# 7.0 features) so I wanted to make it explicit that it's not 6.

https://github.com/irmen/Pyrolite/blob/060bc3c9069cd31560b6da4f67280736fb2cdf63/dotnet/Razorvine.Pyrolite/Tests/Tests.csproj#L7

Reply:
I've updated the language version on all projects to 7.3 now in another commit

Reply:
Great, I have updated the new Benchmarks.csproj to use C# 7.3 as well